data/formats-data.js

```diff
@@ -5367,7 +5367,7 @@ let BattleFormatsData = {
 		doublesTier: "DUU",
 	},
 	emolga: {
-		randomBattleMoves: ["encore", "chargebeam", "uturn", "thunderbolt", "airslash", "roost"],
+		randomBattleMoves: ["encore", "chargebeam", "uturn", "thunderbolt", "voltswitch", "airslash", "roost"],
```

Comment:
It gets U-turn so I don't see the point. A set with two Electric attacks on such a mon makes it more useless.

Reply:
Oops, didn't notice it already had it.